source/clear-linux/tutorials/dlaas-benchmarks.rst

```diff
@@ -0,0 +1,186 @@
+
+.. code-block:: bash
+
+  mkdir -p /opt
+  git clone https://github.com/tensorflow/benchmarks.git /opt/tf-benchmarks
+  cp launcher.py /opt
+  chmod u+x /opt/*
+
+Your entry point then becomes "/opt/launcher.py".
+
+
+This will build an image which can be consumed directly by TFJob from kubeflow.  We are working to create these images as part of our release cycle.
+
+
+Ksonnet
++++++++
+
+Kubeflow uses Ksonnet to manage deployments, so we need to install that before setting up Kubeflow. On |CL|, follow these steps:
+
+.. code-block:: bash
+
+  swupd bundle-add go-basic-dev
+  export GOPATH=$HOME/go
+  export PATH=$PATH:$GOPATH/bin
+  go get github.com/ksonnet/ksonnet
+  cd $GOPATH/src/github.com/ksonnet/ksonnet
+  make install
+
+
+.. For other OS, follow steps in: https://www.kubeflow.org/docs/guides/components/ksonnet/ .
+
+After the ksonnet installation is complete, ensure that binary `ks` is accessible across the environment.
+
+
+Kubeflow
+========
+Once you have k8s running on your nodes, you can setup Kubeflow by following these instructions from their `quick start guide`_.
+
+.. code-block:: bash
+
+  export KUBEFLOW_SRC=$HOME/kflow
+  export KUBEFLOW_TAG=”v0.3.2”
+  export KFAPP=”kflow_app”
+  export K8S_NAMESPACE=”kubeflow”
+  mkdir ${KUBEFLOW_SRC}
+  cd ${KUBEFLOW_SRC}
+  curl https://raw.githubusercontent.com/kubeflow/kubeflow/${KUBEFLOW_TAG}/scripts/download.sh | bash
+  ${KUBEFLOW_SRC}/scripts/kfctl.sh init ${KFAPP} --platform none
+  cd ${KFAPP}
+  ${KUBEFLOW_SRC}/scripts/kfctl.sh generate k8s
+
+Now you have all the required kubeflow packages, and you can deploy the primary one for our purposes: tf-job-operator.
+
+.. code-block:: bash
+
+  kubectl create namespace ${K8S_NAMESPACE}
+  ks env add default --namespace "${K8S_NAMESPACE}"
+  ks apply default -c tf-job-operator
+
+This creates the CustomResourceDefinition(CRD) endpoint to launch a TFJob.
+
+Running the Deep Learning as a Service TFJob
+++++++++++++++++++++++++++++++++++++++++++++
+
+The jsonnet template files for ResNet50 and Alexnet are available in the |CL| Deep Learning Stack repository. Download and copy these files into
+
+.. code-block:: console
+
+  ${KUBEFLOW_SRC}/${KFAPP}/vendor/kubeflow/examples/prototypes/
+
+Next, generate Kubernetes manifests for the workloads and apply them to create and run them using these commands
+
+.. code-block:: bash
+
+  ks generate dlaas-resnet50 dlaasresnet50 --name=dlaasresnet50
+  ks generate dlaas-alexnet dlaasalexnet --name=dlaasalexnet
+  ks apply default -c dlaasresnet50
+  ks apply default -c dlaasalexnet
+
+This will replicate and deploy three test setups in your k8s cluster.
+
+
+Results
+=======
+You will need to parse the logs of the k8s pod to get the performance numbers. The pods will still be around post completion and will be in ‘Completed’ state. You can get the logs from any of the pods to inpsect the benchmark results.
```

Comment:
What will I be parsing for?  Can I use grep to find it by doing a cat | grep command?